src/app.js

```diff
@@ -25,11 +37,30 @@ const quoteData = [
   },
 ];
 
+
 $(document).ready(function() {
   const quotes = new QuoteList(quoteData);
+  const orders = new OrderList();
   const simulator = new Simulator({
     quotes: quotes,
   });
 
+  const quoteListView = new QuoteListView({
+    model: quotes,
+    template: _.template($('#quote-template').html()),
+    tradesTemplate: _.template($('#trade-template').html()),
+    el: 'main'
+  });
+
+  const orderListView = new OrderListView({
+    model: orders,
+    template: _.template($('#order-template').html()),
+    el: 'main'
```

Comment:
Also note that both views represent the **SAME** element in the DOM, this means it's possible they could interfere with each other.  

Instead I would suggest making the `QuoteListView`'s `el` element be `#quotes-container` and the `OrderListView`'s `el` be `#order-workspace`.